arch/arm/configs/siklu_PCB277_V2_defconfig

```diff
@@ -30,12 +30,6 @@ CONFIG_HIGHMEM=y
 # CONFIG_ARM_MODULE_PLTS is not set
 CONFIG_FORCE_MAX_ZONEORDER=14
 CONFIG_CMDLINE="noinitrd console=ttymxc0,115200"
-CONFIG_CPU_FREQ=y
```

Comment:
what will be the CPU frequency if disabling this? We need to run CPU in max. frequency